xml/templates/message_templates/contribution_online_receipt_text.tpl

```diff
@@ -126,14 +126,14 @@
 {/foreach}
 {/if}
 
-{if !( $contributeMode eq 'notify' OR $contributeMode eq 'directIPN' ) and $is_monetary}
+{if ($is_pay_later || $billingName)}
```

Comment:
I think on this one we need to reverse the IF order

Reply:
so we have 2 if statements here L129 an L130 and the elseif in L136 only relates to the if in L130 as far as i can tell